modules/eks/providers.tf

```diff
@@ -8,14 +8,29 @@ provider "aws" {
   }
 }
 
+provider "aws" {
```

Comment:
@osterman you and I talked about this lightly but we didn't come up with an action item I don't believe... it seems funky that we're requiring spotinst for our EKS component. Do we potentially want to pull this out to its own component like `eks-with-spotinst` or a fully separate `spotinst-eks-node-group` component? cc @aknysh